app/models/session.rb

```diff
@@ -4,6 +4,14 @@
 # an applicant that applies to Fall 2018, round 1 applies to the "Fall 2018 Round 1" session.
 class Session < ApplicationRecord
   has_many :positions
+
+  # TODO: Since we're using enums for semester, we might have to change semester to integer for type.
```

Comment:
is this optional or a must do? :/

Reply:
🤔

Reply:
Can you see what it's saved as then?

Reply:
Insertions work for me. Will create ticket to double check if this TODO can be removed